### PR TITLE
Add doc for animal birth reg

### PR DESCRIPTION
**Pull request by VicXue**
----
[AB#17141](https://dev.azure.com/rezaresystems/15020b92-fee6-4202-956f-9036d1e1da84/_workitems/edit/17141)
Notes

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [ ] Links work to navigate documentation
- [ ] Images stored in `_images`
- [ ] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/animal_birth_registration.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+|**Trait Label**|If the event represents a formal trait, this identifies the recording system and trait. |Identifier|
+|**Responsible**|Use if an observation is manually recorded, or an event is carried out or authorised by a person. SHOULD be a person object.    |String |
+|**Remark**|A comment or remark field for additional user-specified information about the event.|String |
+|**Contemporary Group**|For manually recorded events, record any contemporary group code that would affect statistical analysis. |String |
+|**Registration Reason**|Enumeration for registration reason: **Born**, or **Registered** (induct existing animal).|Enumeration|
+|**Location**|An identifier for the location of the Animal Birth Registration. |Identifier|
+|**Remark**|A comment or remark field for additional user-specified information about the event.|String |
+|**Animal**|Unique animal scheme and identifier combination.|Identifier|
+|**Animal Detail**|Core schema for representing animal. [See the Animal definition](/animal). |AnimalDetail|
```
Comment (VicXue) [inline docs/resource-types/animal_birth_registration.md:61]: Not sure how this relative link to Animal should be structured.

**Comment by daniel-buchanan** [comment]
@VicXue this should probably be located at `docs/resource-types/livestock/birth-registration.md`.
Also note that you'll need the following metadata at the top of the file:
```
---

title: Title of the post
menu_order: 1
post_status: publish
post_excerpt: This is a post excerpt
taxonomy:
    category:
        - category-slug-1
        - category-slug-2
    post_tag:
        - tag-1
        - tag-2
custom_fields:
    field1: value 1
    field2: value 2

---
```
You'll need to update it to reflect what the page is supposed to be.

**Comment by daniel-buchanan** [comment]
@VicXue I've updated the formatting - see what you think.

**Comment by VicXue** [comment]
> @VicXue I've updated the formatting - see what you think.

Looks good to me. Does the path need to follow the resource type name? i.e. `/livestock/movement/birth-registration.md` Or it can stay as it is as long as we know the structure?

File: docs/resource-types/animal/birth-registration.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+|**Trait Label**|If the event represents a formal trait, this identifies the recording system and trait. |Identifier|
+|**Responsible**|Use if an observation is manually recorded, or an event is carried out or authorised by a person. SHOULD be a person object.    |String |
+|**Remark**|A comment or remark field for additional user-specified information about the event.|String |
+|**Contemporary Group**|For manually recorded events, record any contemporary group code that would affect statistical analysis. |String |
+|**Registration Reason**|Enumeration for registration reason: **Born**, or **Registered** (induct existing animal).|Enumeration|
+|**Location**|An identifier for the location of the Animal Birth Registration. |Identifier|
+|**Remark**|A comment or remark field for additional user-specified information about the event.|String |
+|**Animal**|Unique animal scheme and identifier combination.|Identifier|
+|**Animal Detail**|Core schema for representing animal. [See the Animal definition](./index.md). |AnimalDetail|
```
Comment (danielR9) [inline docs/resource-types/animal/birth-registration.md:82]:
```suggestion
|**Animal Detail**|Core schema for representing animal. [See the Animal definition](/resource-types/animal/index.md). |AnimalDetail|
```
From looking at the stuff from @daniel-buchanan  I think we might need absolute references

File: docs/resource-types/animal_birth_registration.md (hunk 1 above)
Comment (daniel-buchanan) [reply docs/resource-types/animal_birth_registration.md]: @VicXue the link should be `[See the Animal Definition](/resource-types/animal/index.md)`.

File: docs/resource-types/animal/birth-registration.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    "resourceType": "/livestock/movement/animal-birth-registration",
+    "eventDateTime": "string",
+    "traitLabel": { ... },
+    "responsible": "string",
+    "remark": "string",
+    "contemporaryGroup": "string",
+    "registrationReason": "string",
+    "location": { ... },
+    "animal": { ... },
+    "animalDetail": { ... }
```
[the remark this answers is not in the document]
Comment (VicXue) [reply docs/resource-types/animal/birth-registration.md:63]: Indenting is consistent on my local env and it's not picked up by GIT for some reason. I manually edited it here using Github.

File: docs/resource-types/animal/birth-registration.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+|**Meta**        |Meta data for the resource.                               |Metadata        |
+|**Resource Type**|The fixed discriminator for the Animal Birth Registration resource type. <br/>Value: ***/livestock/movement/animal-birth-registration***    |String|
+|**Event DateTime** |The datetime when the event happened. Shall be UTC format with Z, specified in RFC3339 (see https://ijmacd.github.io/rfc3339-iso8601/ for format guidance).|Date/Time|
+|**Trait Label**|If the event represents a formal trait, this identifies the recording system and trait. |Identifier|
+|**Responsible**|Use if an observation is manually recorded, or an event is carried out or authorised by a person. SHOULD be a person object.    |String |
+|**Remark**|A comment or remark field for additional user-specified information about the event.|String |
+|**Contemporary Group**|For manually recorded events, record any contemporary group code that would affect statistical analysis. |String |
+|**Registration Reason**|Enumeration for registration reason: **Born**, or **Registered** (induct existing animal).|Enumeration|
```
[the remark this answers is not in the document]
Comment (VicXue) [reply docs/resource-types/animal/birth-registration.md:78]: I guess we just keep them here for now. It's how other docs are doing it.

File: docs/resource-types/animal/birth-registration.md (hunk 2 above)
Comment (VicXue) [reply docs/resource-types/animal/birth-registration.md:82]: Where is the reference for this? I was referencing [this part here](https://github.com/Pure-Farming/Documentation/commit/1bf348fc3f25aad5fb0ecd37f30905cf5ef8154b#diff-4d0911d71013f4d13b46164183b83916168aeb4bbf36dcefdc054dafae78667eR37-R44). This md file and the `index.md` for Animal will be stored in the same folder, so I guess we can use the relative path?

Comment (daniel-buchanan) [reply docs/resource-types/animal/birth-registration.md]: @VicXue this has all changed, and I've updated it.

**Review by danielR9 — APPROVED** [review]

File: docs/resource-types/livestock/movements/birth-registrations.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+}
+```
+
+|Response Item   |Description                    |Data Type                    |
+|----------------|-------------------------------|-----------------------------|
+|**Self**        |A link to the specific Animal Birth Registration.         |URL             |
+|**Id**          |The Pure Farming Id of this Animal Birth Registration.    |UUID            |
+|**Meta**        |Meta data for the resource.                               |Metadata        |
+|**Resource Type**|The fixed discriminator for the Animal Birth Registration resource type. <br/>Value: ***/livestock/movement/animal-birth-registration***    |String|
+|**Event DateTime** |The datetime when the event happened. Shall be UTC format with Z, specified in RFC3339 (see https://ijmacd.github.io/rfc3339-iso8601/ for format guidance).|Date/Time|
```
Comment (danielR9) [inline docs/resource-types/livestock/movements/birth-registrations.md:73]: Only suggestion would be to put the url in as a link? - I think this syntax will work?
```suggestion
|**Event DateTime** |The datetime when the event happened. Shall be UTC format with Z, specified in [RFC3339](https://ijmacd.github.io/rfc3339-iso8601/). |Date/Time|
```

**Review by danielR9 — DISMISSED** [review]

**Review by daniel-buchanan — APPROVED** [review]

**Pull request merged**